src/lib/FlightTasks/tasks/Utility/ManualSmoothingZ.hpp

```diff
@@ -64,17 +65,20 @@ class ManualSmoothingZ
 
 
 	/* Getter methods */
-	float getMaxAcceleration() {return _max_acceleration;};
-	ManualIntentionZ getIntention() {return _intention;};
+	float getMaxAcceleration() { return _max_acceleration; }
+	ManualIntentionZ getIntention() { return _intention; }
 
 	/* Overwrite methods:
 	 * Needed if different parameter values than default required.
 	 */
-	void overwriteAccelerationUp(float acc_max_up) {_acc_max_up = acc_max_up;};
-	void overwriteAccelerationDown(float acc_max_down) {_acc_max_down = acc_max_down;};
-	void overwriteJerkMax(float jerk_max) {_jerk_max = jerk_max;};
+	void overwriteAccelerationUp(float acc_max_up) { _acc_max_up.set(acc_max_up); }
```

Comment:
I think these overwritten values are temporary and should not be saved to the parameters in the flash memory right?Is that still the case?

Reply:
yes, it does not write back to the param store